bpm/bonita-core/bonita-process-instance/bonita-process-instance-impl/src/main/resources/org/bonitasoft/engine/core/process/instance/model/impl/hibernate/process.instance.queries.hbm.xml

```diff
@@ -83,6 +83,7 @@
 		WHERE g.logicalGroup4 = :parentProcessInstanceId
 		AND g.name = :name
 		AND g.terminal = FALSE
+		AND g.hitBys NOT LIKE 'FINISH%'
```

Comment:
using "not like" in query is not recommended against sql performance, since a full table scan will be done. 

alternative way could be to add an index on substring(hitBys,0,6) but it's not a smart way  

consider adding a specific flag to identify those gateways